src/jit/codegenlinear.cpp

```diff
@@ -1389,6 +1403,45 @@ void CodeGen::genConsumeOperands(GenTreeOp* tree)
     }
 }
 
+#ifdef FEATURE_HW_INTRINSICS
+//------------------------------------------------------------------------
+// genConsumeHWIntrinsicOperands: Do liveness update for the operands of a GT_HWIntrinsic node
+//
+// Arguments:
+//    node - the GenTreeHWIntrinsic node whose operands will have their liveness updated.
+//
+// Return Value:
+//    None.
+//
+
+void CodeGen::genConsumeHWIntrinsicOperands(GenTreeHWIntrinsic* node)
+{
+    GenTree* op1 = node->gtGetOp1();
+    if (op1 == nullptr)
+    {
+        assert(node->gtGetOp2() == nullptr);
+        return;
+    }
+    if (op1->OperIs(GT_LIST))
```

Comment:
nit: `OperIsList()`?

Reply:
I wouldn't bother too much with this. Hopefully we'll just get rid of `GT_LIST`. I have ~75% of it done, just waiting for intrinsics to finally stabilize so I don't have to keep fixing conflicts.